packages/babel-plugin-transform-react-jsx/test/fixtures/react/should-escape-xhtml-jsxattribute/actual.js

```diff
@@ -0,0 +1,3 @@
+<div id="wôw" />;
+<div id="\w" />;
+<div id="w &lt; w" />;
```

Comment:
would appreciate suggestions for other test-cases, as I think this is the crux of the issue (and was previously untested)